tests/account-factory-wiring.yaml

```diff
@@ -112,8 +112,8 @@ Resources:
               Service: lambda.amazonaws.com
             Action: sts:AssumeRole
       ManagedPolicyArns:
-        - arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole
-        - arn:aws:iam::aws:policy/AdministratorAccess # fixme: least privilege
+        - !Sub arn:${AWS::Partition}:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole
```

Comment:
AFAIK, we don't need to touch occurrences outside the `templates` folder, since we only sync the `templates` folder to the quick start repo.

Reply:
> AFAIK, we don't need to touch occurrences outside the `templates` folder, since we only sync the `templates` folder to the quick start repo.

I figured since it was suggested to be 'good practice' it should be generally applied, which is why i just included it in the tests folder as well.